textattack/commands/attack/attack_command.py

```diff
@@ -81,6 +81,13 @@ def register_subcommand(main_parser: ArgumentParser):
             help="A directory to output results to.",
         )
 
+        parser.add_argument(
+            "--txt-filename",
```

Comment:
Maybe instead of adding txt/csv filename arguments separate from out-dir, we can do something like turn out-dir an --out-file argument that performs the functionality of out-dir if it ends in /, but otherwise is treated as a filename rather than directory. And maybe we can just change the extension based on txt vs. csv?

Reply:
Hey Eli, I've merged `--out-dir`, `--txt-filename`, and `--csv-filename` into `--out-file-txt` and `--out-file-csv`

when `--out-file-txt` and `--out-file-csv` arguments don't end with "/", they are used to choose specific filename

when they do end with "/", they work as `--out-dir`

is that what you mean?  please let me know!

Reply:
(Replied in the general comment in my below review)